src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -154,5 +154,73 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
                     )
     endfunction()
 
+    set(PICO_CYW43_DRIVER_CURRENT_PATH ${CMAKE_CURRENT_LIST_DIR})
+    pico_register_common_scope_var(PICO_CYW43_DRIVER_CURRENT_PATH)
+
     pico_promote_common_scope_vars()
+
+    function(pico_use_partition_firmware TARGET)
```

Comment:
I think in the future as mentioned elsewhere we might want to allow more params to configure how we do things (partition size/loc, versioninfo, partition ID) but this is fine too add via CMake args later

That said, the name in the global namespace is a bit non-specific. perhaps `pico_configure_wifi_firmware_partition' (or select instead of configure

Reply:
Maybe `pico_use_wifi_firmware_partition`?